### ISSUE TITLE
[customer]remoteshell failed in line 488 in statelite sles

### ISSUE THREAD

**Issue by bybai** [xcat2/xcat-core#2726]
From Gilad Berman, lenovo:
In the remoshell script you are enabling ssh service (line 488) – 
```
else

    #service sshd restart

    # sshd is not enabled on SLES 12 by default

    # does not hurt anything to re-enable if it is enabled already

    enableservice sshd

    restartservice sshd

fi
```
For statelite, the /etc/systemd/system..  is a read only file system, of course. This makes the script fail with error message. 
One obvious solution is to place it in tmpfs (which we did), but ideally, we don’t want to do that, there is no reason for it to be in tmpfs. 
I guess a simple “if” will solve it 😊

**Comment by daniceexi** [comment]
@bybai what's your idea for this change? disable the action **enable the sshd** should be accepted for diskless/stetelite.

**Comment by bybai** [comment]
Hi @daniceexi ， There is comments in the remoteshell here:
```
else
    #service sshd restart
    # sshd is not enabled on SLES 12 by default
    # does not hurt anything to re-enable if it is enabled already
    enableservice sshd
    restartservice sshd
fi 
```
I think there is "sshd is not enabled on SLES 12 by default",  so I will have a test in sles12sp2 to look into it in 2.13.4 sprint1, then we can decide if disable the action `enable the sshd` in all the OS.

**Comment by daniceexi** [comment]
@bybai I think the problem is the **remoteshell** script is common for both diskless and diskful. I do think 'enable sshd' is necessary for diskful. So we can just disable the 'sshd enablement' for diskless and statelite.

**Comment by bybai** [comment]
@daniceexi , that's fine, thank you.

**Comment by bybai** [comment]
fixed in #3017

**Comment by bybai** [comment]
fixed in #3027

**Comment by bybai** [comment]
Verified in auto test env.
